src/javascript/_common/base/client_base.js

```diff
@@ -451,6 +429,42 @@ const ClientBase = (() => {
         return options_blocked_countries.includes(country);
     };
 
+    const isHighRisk = () => {
```

Comment:
Let's add `risk_classification` into an account as well, currently the function doesn't cover all high risk cases.

Reply:
Added all checks to be same as in deriv-app for high risk and low risk :)